src/debugger/types.js

```diff
@@ -9,9 +9,22 @@
 
 /* @flow */
 
-export type BreakpointCommandArguments = {
-  kind: string,
+export type DebuggerRequest = {
```

Comment:
I think you will need a DebuggerResponse message sometime later. 
When that happens, you may want to take out "requestID"(or event kind) out and put inside DebuggerRequest/DebuggerResponse.